src/core/thread/mle.cpp

```diff
@@ -637,6 +641,17 @@ void Mle::SetAssignLinkQuality(const Mac::ExtAddress aMacAddr, uint8_t aLinkQual
     }
 }
 
+uint8_t Mle::GetRouterSelectionJitter(void) const
+{
+    return mSetRouterJitter ? mRouterJitter : mRouterSelectionJitterTimeout;
+}
+
+void Mle::SetRouterSelectionJitter(uint8_t aRouterJitter)
+{
+    mSetRouterJitter = true;
```

Comment:
Using a different value for `kRouterSelectionJitter` should only happen during testing.  Is there a test case that requires having different values of `kRouterSelectionJitter` in `HandleChildIdResponse()` and `HandleAdvertisement()`?